src/models/multiBarChart.js

```diff
@@ -430,6 +430,9 @@ nv.models.multiBarChart = function() {
         rightAlignYAxis: {get: function(){return rightAlignYAxis;}, set: function(_){
             rightAlignYAxis = _;
             yAxis.orient( rightAlignYAxis ? 'right' : 'left');
+        }},
+        disableBarColor:  {get: function(){return multibar.barColor === undefined;}, set: function(_){
```

Comment:
Adding the option here is unnecessary since it inherits the option from multibar already below.